sdk/core/core/internal/az_http_internal.h

```diff
@@ -170,6 +172,15 @@ az_http_request_set_query_parameter(_az_http_request* p_request, az_span name, a
 AZ_NODISCARD az_result
 az_http_request_append_header(_az_http_request* p_request, az_span key, az_span value);
 
+/**
+ * @brief Validate HTTP retry options.
+ *
+ * @param retry_options HTTP retry options to validate.
+ * @return true if return options are valid, false if invalid.
+ */
+AZ_NODISCARD bool az_http_policy_retry_options_validate(
```

Comment:
I don't think we need this function

Reply:
https://github.com/Azure/azure-sdk-for-c/pull/413#discussion_r387288780

Reply:
Done :)